ui/articles-aem-renderer/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
         <activity
             android:name="org.cru.godtools.articles.aem.activity.AemArticleActivity"
             android:parentActivityName="org.cru.godtools.article.activity.ArticlesActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|uiMode|screenSize"
```

Comment:
use retainInstance on the fragment instead

Reply:
example logic here: https://github.com/CruGlobal/android-thekey/blob/master/thekey-view-dialog-fragment/src/main/java/me/thekey/android/view/dialog/LoginDialogFragment.java#L55

Reply:
it's older logic, but I'd prefer we avoid overriding configuration changes as much as possible